plugins/plugin-site/src/components/PluginReleases.jsx

```diff
@@ -53,7 +53,7 @@ function PluginReleases({pluginId, versions}) {
                                         <a href={release.htmlURL} title="See the release on GitHub" className="github-icon d-flex"><ion-icon name="logo-github" /></a>
                                     </>)}
                                     <a className="anchor after" href={`#version_${version.version}`}>
-                                        <ion-icon name="link-outline"></ion-icon>
+                                        <ion-icon name="link-outline"/>
```

Comment:
I believe web components must have closing tags, not self closing.

Reply:
You're right, self-closing custom elements are still not valid HTML (as far as I understand https://github.com/WICG/webcomponents/issues/624 ), but 
* ESlint prefers self-closing tags and we're using it this way for all icons in the repo
* This is .jsx so HTML parsing restrictions do not apply